tests/ui/invalid_result_conversion.stderr

```diff
@@ -6,8 +6,8 @@ error[E0277]: the trait bound `PyErr: From<MyError>` is not satisfied
    |
    = help: the following other types implement trait `From<T>`:
              <PyErr as From<PyBorrowError>>
-             <PyErr as From<PyBorrowMutError>>
              <PyErr as From<std::io::Error>>
+             <PyErr as From<PyBorrowMutError>>
```

Comment:
From some quick bisecting this diff appears to have been caused by adding `PyTupleMethods` to the prelude, which doesn't make any sense to me. Still, I don't think there's any particular problem with the order here changing (it does between Rust versions anyway).

Reply:
Yes, I encountered this as well I think the churn is to be expected if we change the input set of types for the compiler. (Using an order that is stable w.r.t. this like alphabetic seems unergonomic for the developer.)